index.md

```diff
@@ -172,11 +175,41 @@ Authors should note whether any funding could be perceived as a conflict of inte
 
 ## Licensing 
 
-LiveCoMS does not require any copyright transfer to the journal.  
-Instead, LiveCoMS requires articles use a license which allows LiveCoMS to distribute the articles freely consistent with Open Access requirements. 
-We recommend that all submissions to LiveCoMS be released by the author under a Creative Commons By Attribution license version 4.0; however, other licenses may be acceptable with justification.
-
-## Policy on Prior Publication
+LiveCoMS does not request or allow any copyright transfer.
+
+However, in order to submit to LiveCoMS, authors must provide, at
+minimum, a license for LiveCoMS to publish the article and distribute
+it free of charge. We recommend that the authors release the article
+under an open source license such as [Creative Commons
+  Attribution](https://creativecommons.org/licenses/by/4.0/}{Creative Commons
+  Attribution) (also known as CC-BY) releasing the document for anyone to copy and
+redistribute the material in any medium or format, and remix,
+transform, and build upon the material for any purpose, even
+commercially.  Making it available to all of course makes it possible
+for LiveCoMS to publish it, and for the community to edit and
+contribute.  We highly recommend the CC-BY license in order to ensure
+your work can reach and help the broadest audience possible, and
+suggest that when considering the appropriate license you [read this analysis](http://openaccess.ox.ac.uk/2013/06/13/cc-by-what-does-it-mean-for-scholarly-articles-3/).
+
+Other more restrictive licenses may be permissible as long as LiveCoMS
+has the permission to publish and excerpt from the document; this
+might be required if someone other than the authors has some rights to
```

Comment:
maybe clarify "this might be required..." as "another license might be needed..."

Reply:
done